hyperspy_swift_library/__init__.py

```diff
@@ -69,11 +69,22 @@ def __init__(self, workspace_dir):
             di.read_properties() for di in self._data_items]
 
     def get_data_items(self):
-        """ Returns DataFrame if Pandas is installed containing the data_items properties in a Nion Library
-        Returns _data_items_properties is panda is not installed
+        """	
+        Parameters
+        ----------	
+        workspace_dir : string
+                        path to Nion Swift library
+        Returns
+        ----------
+
+        DataFrame : Pandas DataFrame containing data_items properties if Pandas is intalled
+                    _data_items_properties if Pandas is not installed
```

Comment:
typo: installed

Also, instead of I would suggest:

[...] if Pandas is installed, a dictionary otherwise containing the same data otherwise.